webpack/ForemanTasks/Components/TasksTable/TasksTableConstants.js

```diff
@@ -0,0 +1,8 @@
+import { getControllerSearchProps } from 'foremanReact/constants';
+
+export const TASKS_CONTROLLER = 'tasks';
+export const TASKS_SUCCESS = 'TASKS_SUCCESS';
+export const SET_SORT_TASKS_TABLE = 'SET_SORT_TASKS_TABLE';
+export const SET_PAGINATION_TASKS_TABLE = 'SET_PAGINATION_TASKS_TABLE';
```

Comment:
The convention is to first have the base name, e.g. `TASKS_TABLE_SET_SORT`, `TASKS_TABLE_SET_PAGINATION`